File: after phase gate later edits updates  all tasks completing/ProductsCheckout/Checkout.py
from datetime import date, datetime

# Collect cashier and customer info
cashier_name = input("Enter cashier's name:\n")
customer_name = input("Enter customer's name:\n")
discount_rate = float(input("Enter discount percentage to give customer:\n"))

items = []
quantities = []
unit_prices = []

add_more_items = True

while add_more_items:
    item = input("Enter product bought:\n")
    quantity = int(input("Enter quantity:\n"))
    price = int(input("Enter price per unit:\n"))

    items.append(item)
    quantities.append(quantity)
    unit_prices.append(price)

    response = input("Add another item? Type YES to continue or type anything else to stop:\n").upper()
    add_more_items = response == "YES"

def display_invoice(customer_name, cashier_name, discount_rate, items, quantities, unit_prices):
    title = """
SEMICOLON STORES
MAIN BRANCH
LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.
TEL: 03293828343
    """
    print("\n" + title)
    print("Date:", date.today())
    print("Time:", datetime.now().strftime("%H:%M:%S"))
    print("Cashier:", cashier_name)
    print("Customer:", customer_name)
    print()

    print("Item\tQty\tPrice/Unit\tTotal")
    print("---")

    bill_grand_total = 0
    for i in range(len(items)):
        total = quantities[i] * unit_prices[i]
        bill_grand_total += total
        print(items[i], "\t", quantities[i], "\tNGN", unit_prices[i], "\tNGN", total)

    vat = bill_grand_total * 0.075
    discount = bill_grand_total * (discount_rate / 100)
    final_total = bill_grand_total + vat - discount

    print("-----")
    print("BILL GRAND TOTAL:", "NGN", bill_grand_total)
    print("VAT @ 7.5%:", "NGN", round(vat, 2))
    print("Discount:", "NGN", round(discount, 2))
    print("TOTAL AFTER DISCOUNT & VAT:", "NGN", round(final_total, 2))
    print("\nTHANK YOU FOR YOUR PATRONAGE!")

display_invoice(customer_name, cashier_name, discount_rate, items, quantities, unit_prices)

File: after phase gate later edits updates  all tasks completing/ProductsCheckout/Checkout.js
const prompt = require('prompt-sync')();

// Collect cashier and customer info
const cashierName = prompt("Enter cashier's name:\n");
const customerName = prompt("Enter customer's name:\n");
const discountRate = parseFloat(prompt("Enter discount percentage to give customer:\n"));

const items = [];
const quantities = [];
const unitPrices = [];

let addMoreItems = true;

while (addMoreItems) {
  const item = prompt("Enter product bought:\n");
  const quantity = parseInt(prompt("Enter quantity:\n"));
  const price = parseInt(prompt("Enter price per unit:\n"));

  items.push(item);
  quantities.push(quantity);
  unitPrices.push(price);

  const response = prompt("Add another item? Type YES to continue or type anything else to stop:\n").toUpperCase();
  addMoreItems = response === "YES";
}

// Display invoice
function displayInvoice(customerName, cashierName, discountRate, items, quantities, unitPrices) {
  const title = `
SEMICOLON STORES
MAIN BRANCH
LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.
TEL: 03293828343
  `;

  console.log("\n" + title);
  console.log("Date:", new Date().toLocaleDateString());
  console.log("Time:", new Date().toLocaleTimeString());
  console.log("Cashier:", cashierName);
  console.log("Customer:", customerName);
  console.log();

  console.log("Item\t\tQty\tPrice/Unit\tTotal");
  console.log("---");

  let billGrandTotal = 0;
  for (let i = 0; i < items.length; i++) {
    const total = quantities[i] * unitPrices[i];
    billGrandTotal += total;
    console.log(`${items[i]}\t\t${quantities[i]}\tNGN${unitPrices[i]}\t\tNGN${total}`);
  }

  const vat = billGrandTotal * 0.075;
  const discount = billGrandTotal * (discountRate / 100);
  const finalTotal = billGrandTotal + vat - discount;

  console.log("-----");
  console.log(`BILL GRAND TOTAL:\tNGN${billGrandTotal}`);
  console.log(`VAT @ 7.5%:\t\tNGN${vat}`);
  console.log(`Discount:\t\tNGN${discount}`);
  console.log(`TOTAL AFTER DISCOUNT & VAT:\tNGN${finalTotal}`);
  console.log("\nTHANK YOU FOR YOUR PATRONAGE!");
}

displayInvoice(customerName, cashierName, discountRate, items, quantities, unitPrices);
